desiregallery/apps/gallery/templates/gallery/base.html

```diff
@@ -2,19 +2,27 @@
 <!DOCTYPE html>
```

Comment:
Attach the footer to the bottom

Reply:
Solution:
https://www.freecodecamp.org/news/how-to-keep-your-footer-where-it-belongs-59c6aa05c59c/